packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -146,22 +146,4 @@ describe('ChoiceGroup', () => {
     expect(extraAttributeGetter(0)).toEqual('auto1');
     expect(extraAttributeGetter(1)).toBeNull();
   });
-
-  // By default, we need to assign the role 'application' on the containing div
-  // because JAWS doesn't call OnKeyDown without this role
-  it('has role="application" by default on the containing element', () => {
-    const choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} />);
-    const choiceGroupEl: Element = choiceGroup.getDOMNode();
-    const role = choiceGroupEl.getAttribute('role');
-
-    expect(role).toEqual('application');
-  });
-
-  it('has role attribute that can be omitted', () => {
-    const choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} role="" />);
-    const choiceGroupEl: Element = choiceGroup.getDOMNode();
-    const role = choiceGroupEl.getAttribute('role');
-
-    expect(role).toEqual('');
-  });
```

Comment:
I would _expect_ these tests to be refactored to:

1. A test which allows for user override of `role` and asserts it is present in the DOM.
1. A test which omits `role` and asserts `role` is not present in the DOM.